src/components/layers-panel/layers-panel.tsx

```diff
@@ -170,6 +172,21 @@ export const LayersPanel = ({
   onPointToLayer,
 }: LayersPanelProps) => {
   const layout = useAppLayout();
+  const location = useLocation();
+
+  const bookmarksPageId = useMemo(() => {
+    switch (location.pathname) {
```

Comment:
I would send the id from the page via props

Reply:
Done